frontend/src/common/hooks/useCategoryFilter/common/utils.ts

```diff
@@ -0,0 +1,114 @@
+import { Filters } from "react-table";
```

Comment:
The functionality in this file was moved mostly unchanged from useCategoryFilter. I have marked the changes below.